src/ensembl/src/content/app/entity-viewer/gene-view/components/collapsed-exons-image/CollapsedExonsImage.tsx

```diff
@@ -76,28 +71,22 @@ export const ExonsImage = (props: ExonsImageProps) => {
 const ExonsImageWithData = (props: ExonsImageWithDataProps) => {
   const { transcript } = props;
 
-  if (!transcript.cds) {
+  if (!transcript.product) {
```

Comment:
We may have discussed this, but aren't you making sure that this component will be created only for transcripts that do have products?

Reply:
Yes sorry, I missed this one out. Will make the change.

Reply:
✔️